ecsclient/baseclient.py

```diff
@@ -109,7 +109,7 @@ def _fetch_headers(self):
         headers = {'Accept': 'application/json',
                    'Content-Type': 'application/json',
                    'x-sds-auth-token': token}
-        if self.override_header != None:
+        if self.override_header is not None:
```

Comment:
Should we be using a double quote around the string

```suggestion
        if self.override_header != "None":
```